### PR TITLE
Adds baseUrl as a user input parameter while init API call

### PR DESCRIPTION
**Pull request by ChetanGN**

### CODE REVIEW

File: src/RazorpayClient.cs [hunk 1]
```diff
@@ -1,233 +1,248 @@
...
+        private Addon addon = null;
+        private Plan plan = null;
+        private Subscription subscription = null;
+        private VirtualAccount virtualaccount = null;
+
+        
+        public RazorpayClient(string key, string secret)
+        {
+            RazorpayClient.Key = key;
+            RazorpayClient.Secret = secret;
```
Comment (sunnyagg) [inline src/RazorpayClient.cs]: initialize BaseUrl as well

File: src/RazorpayClient.cs [hunk 2]
```diff
@@ -1,233 +1,248 @@
...
+        }
+        
+        public RazorpayClient(string key, string secret, string baseUrl)
```
Comment (sunnyagg) [inline src/RazorpayClient.cs]: make baseUrl first param

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Usage
 -----
 ### Initialize
 ```cs
-RazorpayClient client = new RazorpayClient(key, secret);
+RazorpayClient client = new RazorpayClient(key, secret, baseUrl);
```
Comment (akashsharma95) [inline README.md]: Should specify that here baseUrl is an optional parameter

File: src/RazorpayClient.cs (hunk 1 above)
Comment (ChetanGN) [reply src/RazorpayClient.cs]: Already initialized it in another constructor.

File: README.md (hunk 3 above)
Comment (ChetanGN) [reply README.md]: Ok, I will update.

File: net45/RazorpayClient45.sln [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (akashsharma95) [inline net45/RazorpayClient45.sln]: remove this file

File: src/RazorpayClient.cs [hunk 5]
```diff
@@ -1,233 +1,253 @@
...
-        }
-    }
-}
+using System.Collections.Generic;
+
+namespace Razorpay.Api
+{
+    public class RazorpayClient
+    {
+        const string CurrentVersion = "3.0.0";
```
Comment (akashsharma95) [inline src/RazorpayClient.cs]: Increment version to `3.0.1`

**Pull request merged**